runner.py

```diff
@@ -78,6 +78,14 @@ def parse_args():
                         default=False, action='store_true',
                         help="Board will not be flashed by runner.")
 
+    # Add alternative option "--long-test" "" for long-test argument, which is False
+    # Needed for proper disabling long tests in gh actions
+    # because of problem with passing empty argument through gh workflows
+    parser.add_argument("--long-test",
+                        type=bool,
+                        nargs='?', default=False, const=True,
+                        help="Long tests will be run")
```

Comment:
Can't we build some primitive ternary operator in github actions? Something like ${{ inputs.long_tests && '--long-test' || '' }}